LoggerInterface/src/FileLogger.java

```diff
@@ -0,0 +1,13 @@
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileLogger implements Logger {
+    @Override
+    public void log(String messeage) {
+        try (FileWriter writer = new FileWriter("C:/log.txt")) {
+            writer.write(messeage);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+}
```

Comment:
The default location can be the path you have given, but what if I want that path + file name to be customizable?